python/cudf/cudf/utils/ioutils.py

```diff
@@ -772,6 +772,29 @@
 1  456 2018-11-14T12:35:01.000 5784
 2  789 2018-11-15T18:02:59.000 6117
 
+Read ``hexadecimal values`` from a csv file as integer column with cudf
```

Comment:
Can you also add a note or example about how `0x...` formatted numbers are parsed into ints if dtype inference is on?